torch/cuda/__init__.py

```diff
@@ -352,6 +352,33 @@ def __init__(self, obj):
         super().__init__(idx)
 
 
+
+def _get_device(device: Union[int, str, torch.device] = 'cuda') -> torch.device:
```

Comment:
Why does this have a default?

Reply:
It's used as a utility function in many functions that omit device arg at callsite, e.g. `torch.cuda.get_rng_state()` is supposed to return rng state on current device.